packages/components/src/VirtualizedList/CellTitle/CellTitle.test.js

```diff
@@ -38,6 +38,36 @@ describe('CellTitle', () => {
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
 
+	it('should render without active class if no onClick on the title', () => {
+		// given
+		const columnData = {
+			id: 'my-title',
+			displayModeKey: 'displayMode',
+			onEditCancel: jest.fn(),
+			onEditSubmit: jest.fn(),
+		};
+		const rowData = {
+			id: 1,
+			displayMode: 'text',
+			icon: 'talend-file-o',
+			title: 'my awesome title',
+		};
+
+		// when
+		const wrapper = shallow(
+			<CellTitle
+				cellData={'my awesome title'}
+				columnData={columnData}
+				getComponent={jest.fn()}
+				rowData={rowData}
+				rowIndex={1}
+			/>,
+		);
+
+		// then
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
Can you check with a selector and a real expectation ?
Because the snapshots are lot's of text hard to read (hard to say what we looking for) when this is not the shape which is tested. And without a snapshot, when there is an other change in the component, we don't have to check all the changed snapshots, so less risk of regressions.

Even I think there should be 2 tests : with and without onClick/active.